acos_client/tests/unit/v30/test_slb_template_persistence.py

```diff
@@ -137,17 +147,45 @@ def test_slb_template_persistence_search(self):
 
     @responses.activate
     def test_slb_template_persistence_search_not_found(self):
-        responses.add(responses.POST, AUTH_URL, json={'authresponse': {'signature': 'foo', 'description': 'bar.'}})
-        url = '{}test1'.format(SLB_URL)
+        responses.add(responses.POST, AUTH_URL, json=self.get_auth_response())
+        url = self.get_url(T_NAME)
         json_response = {
             'response': {'status': 'fail', 'err': {
                 'code': 1023460352, 'from': 'CM', 'msg': 'Object specified does not exist (object: cookie)'}}
         }
         responses.add(responses.GET, url, json=json_response, status=404)
 
         with self.assertRaises(acos_errors.NotFound):
-            self.client.slb.template.cookie_persistence.get('test1')
+            self.client.slb.template.cookie_persistence.get(T_NAME)
 
         self.assertEqual(len(responses.calls), 2)
         self.assertEqual(responses.calls[1].request.method, responses.GET)
         self.assertEqual(responses.calls[1].request.url, url)
+
+    @responses.activate
+    def test_slb_template_persistence_create_domain(self):
+        # NOTE: Create method tries GET request to see if template exists before using POST request to create template.
+        responses.add(responses.POST, AUTH_URL, json=self.get_auth_response())
+
+        get_url = self.get_url(T_NAME)
+        get_json_response = {
+            'response': {'status': 'fail', 'err': {
+                'code': 1023460352, 'from': 'CM', 'msg': 'Object specified does not exist (object: cookie)'}}
+        }
+        responses.add(responses.GET, get_url, json=get_json_response, status=404)
+        post_url = SLB_URL
+        post_json_response = {
+            'cookie': {'name': T_NAME, 'dont-honor-conn-rules': 0, 'insert-always': 0, 'encrypt-level': 1,
+                       'encrypted': 'foo', 'cookie-name': 'sto-id', 'path': '/', 'pass-thru': 0, 'secure': 0,
+                       'httponly': 0, 'match-type': 0, 'uuid': '40143610-6a61-11e8-9bf6-c3c3b3edcd4e'}
+        }
+        responses.add(responses.POST, post_url, json=post_json_response, status=200)
+
+        resp = self.client.slb.template.cookie_persistence.create(T_NAME, T_DOMAIN)
+
+        self.assertIsNone(resp)
+        self.assertEqual(len(responses.calls), 3)
```

Comment:
Missing a test for the response data: `self.assertEqual(resp, post_json_response)`